ale_linters/elm/make.vim

```diff
@@ -1,6 +1,15 @@
-" Author: buffalocoder - https://github.com/buffalocoder
+" Author: buffalocoder - https://github.com/buffalocoder, soywod - https://github.com/soywod
 " Description: Elm linting in Ale. Closely follows the Syntastic checker in https://github.com/ElmCast/elm-vim.
 
+call ale#Set('elm_make_executable', 'elm-make')
+call ale#Set('elm_make_use_global', 1)
```

Comment:
Change the default to `0`, to match the other linters. We should always prefer using a version more appropriate for a project by default.

Reply:
Done

Reply:
I don't see the changes yet. You might need to push a commit.

Reply:
It's just for me, to not forget any task. It's done in local, I'll push soon